packages/app/package.json

```
@@ -73,7 +73,7 @@
   },
   "sdkVersions": {
     "ios": {
-      "firebase": "10.19.0",
+      "firebase": "10.20.0",
```

reverting this change - I control the dependencies with PRs that I do on their own with testing - and I just updated this in #7585 - also it appears 10.21.0 will be the first release with the PR you really need...

```suggestion
      "firebase": "10.19.0",
```